src/components/Navigation.vue

```diff
@@ -223,5 +228,23 @@ to {
 :deep(.settings-button) {
 	opacity: .7 !important;
 	font-weight: bold !important;
+	z-index: 1;
+}
+.outbox-border {
+	border-top: 1px solid var(--color-background-darker);
+}
+.outbox {
+	margin-left: 8px;
+	height: 54px;
+}
+:deep(.app-navigation-entry:active),
```

Comment:
I have the feeling we are using deep styles a bit too often. Is there no other way? Should we add a slot to the component that allows it to render without a background?

Reply:
i fixed the code a bit, i only left the .active state to be transparent as it aesthetically nicer.